ct_configrecorder_override_consumer.py

```diff
@@ -60,62 +66,68 @@ def assume_role(account_id, role='AWSControlTowerExecution'):
                     sts_session = boto3.Session(
                         aws_access_key_id=response['Credentials']['AccessKeyId'],
                         aws_secret_access_key=response['Credentials']['SecretAccessKey'],
-                        aws_session_token=response['Credentials']['SessionToken']   )
+                        aws_session_token=response['Credentials']['SessionToken'])
 
                     return sts_session
-            except ClientError as exe:
+            except botocore.exceptions.ClientError as exe:
                 logging.error('Unable to assume role')
                 raise exe
 
-
         sts_session = assume_role(account_id)
         logging.info(f'Printing STS session: {sts_session}')
 
-        #Lets use the session and create a client for configservice
-        configservice = sts_session.client('config',region_name=aws_region)
+        # Use the session and create a client for configservice
+        configservice = sts_session.client('config', region_name=aws_region)
 
-        #lets describe for configguration recorder
+        # Describe configuration recorder
         configrecorder = configservice.describe_configuration_recorders()
-        logging.info(f'Existing Configuration Recorder :' ,configrecorder)
+        logging.info(f'Existing Configuration Recorder :', configrecorder)
 
-        #ControlTower created configuration recorder with name "aws-controltower-BaselineConfigRecorder" and we will update just that
+        # ControlTower created configuration recorder with name "aws-controltower-BaselineConfigRecorder" and we will update just that
         try:
             role_arn = 'arn:aws:iam::' + account_id + ':role/aws-controltower-ConfigRecorderRole'
-            CONFIG_RECORDER_RESOURCE_STRING = os.getenv('CONFIG_RECORDER_RESOURCE_LIST')
-            CONFIG_RECORDER_RESOURCE_LIST = CONFIG_RECORDER_RESOURCE_STRING.split(',')
+
+            CONFIG_RECORDER_EXCLUSION_RESOURCE_STRING = os.getenv('CONFIG_RECORDER_EXCLUDED_RESOURCE_LIST')
```

Comment:
May be worth renaming that ENV variable to match the variable here.  Not sure if it's a typo or it was intended.
```suggestion
            CONFIG_RECORDER_EXCLUSION_RESOURCE_STRING = os.getenv('CONFIG_RECORDER_EXCLUSION_RESOURCE_LIST')
```

Reply:
@word this was intended, please see line 82 in the corresponding PR https://github.com/scalefactory/aws-foundations/pull/264/files

Reply:
all good